### PR TITLE
Update User Guide to describe --latest tag in publish-RepoSense

### PR DESCRIPTION
**Pull request by jedkohjk**
Related to #2208 and addresses comments by @damithc in reposense/publish-RepoSense#12

### Proposed commit message
```
Update User Guide to describe --latest tag in publish-RepoSense
```

### CODE REVIEW

File: docs/ug/runSh.md [hunk 1]
```diff
@@ -28,4 +28,5 @@ Depending on which version you wish to use for report generation, add one of the
 * `--release`: Use the latest release (Stable)
 * `--master`: Use the latest version of the master branch
 * `--tag TAG` (e.g. `--tag v1.6.1`): Use the version identified by the Git tag given
+* `--latest TAG`: Gets the latest version with the given version prefix (e.g. `--latest v1.6` gets `v1.6.1`)
```
Comment (damithc) [inline docs/ug/runSh.md:31]: How about the following?

```suggestion
* `--latest TAG_PREFIX`: Gets the latest version with the given tag prefix (e.g. `--latest v1.6` can get `v1.6.1`)
```

Comment (jedkohjk) [reply docs/ug/runSh.md]: Sure, committed

**Review by gok99 — APPROVED** [review]
LGTM!

File: docs/ug/runSh.md [hunk 2]
```diff
@@ -28,4 +28,5 @@ Depending on which version you wish to use for report generation, add one of the
 * `--release`: Use the latest release (Stable)
 * `--master`: Use the latest version of the master branch
 * `--tag TAG` (e.g. `--tag v1.6.1`): Use the version identified by the Git tag given
+* `--latest TAG_PREFIX`: Gets the latest version with the given tag prefix (e.g. `--latest v1.6` can get `v1.6.1`)
```
Comment (ckcherry23) [inline docs/ug/runSh.md:31]: I've modified the phrasing to match the rest of the flags.
```suggestion
* `--latest TAG_PREFIX`: Use the latest version with the given tag prefix (e.g. `--latest v1.6` can use `v1.6.1`)
```

Comment (jedkohjk) [reply docs/ug/runSh.md]: Committed

**Review by ckcherry23 — APPROVED** [review]
LGTM!

**Pull request merged**